app/src/app/components/subheading/subheading.component.html

```diff
@@ -1,3 +1,8 @@
+<!-- view stix id -->
+<app-stixid-property class="grow-to-row" [config]="{
+    object: config.object
+}"></app-stixid-property>
```

Comment:
Similar to my comment above, I'd recommend adding `*ngIf="config.object['stixID']"` here. This ensures that the component is only rendered when a STIX ID is present, avoiding unnecessary rendering